src/Symbolica.Extensions.Configuration.FSharp/Decode.fs

```diff
@@ -1,25 +1,65 @@
 namespace Symbolica.Extensions.Configuration.FSharp
 
+/// <summary>
+/// A function that takes a value and returns a <see cref="Binder"/> which evaluates to <c>Success</c> if the
+/// value can be decoded otherwise <c>Failure</c>.
+/// </summary>
 type Decoder<'a> = string -> Binder<'a>
 
+/// Contains decoders for common types and combinators functions for building new decoders.
```

Comment:
Should it be "combinator functions"? You may be in French mode.

Reply:
It should. Nice spot.